yarn.lock

```diff
@@ -0,0 +1,11303 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't understand why we need yarn here, also if you are making a change in one PR please don't make the same change in another PR.

Reply:
There is no **need**. It's often optional which package manager one is using and I'm pushing this as it's a best practice to track **lock** files in git.
Please have a look [here](https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files/44904494)

Reply:
we already have package-lock.json. please remove yarn as it's not used in this repo.